crates/sui-core/src/authority/shared_object_version_manager.rs

```diff
@@ -177,6 +198,12 @@ impl SharedObjVerManager {
         )?;
         let mut assigned_versions = Vec::new();
         for assignable in assignables {
+            if matches!(assignable, Schedulable::AccumulatorSettlement(_, _))
+                && !epoch_store.accumulators_enabled()
+            {
+                continue;
+            }
```

Comment:
This should not happen, though

Reply:
you're right, in an earlier version of the code i did the gating here. replaced with an assert